src/main/java/seedu/jarvis/model/cca/CcaList.java

```diff
@@ -30,11 +34,30 @@ public CcaList(){
      *
      * @param internalCcaList
      */
-    public CcaList(ArrayList<Cca> internalCcaList) {
+    public CcaList(ObservableList<Cca> internalCcaList) {
```

Comment:
AB3's current implementation does not contain any constructor for their `PersonList`. The constructor may be redundant.